src/mono/wasm/testassets/Wasm.Buid.Tests.Programs/UnmanagedCallback.cs

```diff
@@ -6,13 +6,24 @@ public unsafe partial class Test
     public unsafe static int Main(string[] args)
     {
         ((IntPtr)(delegate* unmanaged<int,int>)&Interop.Managed8\u4F60Func).ToString();
-
         Console.WriteLine($"main: {args.Length}");
         Interop.UnmanagedFunc();
+
         return 42;
     }
 }
 
+namespace Conflict.A {
+    file class Interop {
+        [UnmanagedCallersOnly(EntryPoint = "ConflictManagedFunc")]
+        public static int Managed8\u4F60Func(int number)
```

Comment:
Please add comment explaining what this is testing.
I would like to see UTF characters with surrogates.

Reply:
I've added surrogate tests to the other test in this

Reply:
So, I forgot C# identifiers have to be (effectively) unicode letters so the test change wasn't valid.  I adjusted it slightly so that it will compile.

Reply:
and roslyn still doesn't accept that in identifiers even though it should be valid afaict